deisctl/Makefile

```diff
@@ -32,14 +32,16 @@ install:
 	godep go install -v .
 
 setup-root-gotools:
-	sudo GOPATH=/tmp/tmpGOPATH go get -u -v code.google.com/p/go.tools/cmd/cover
-	sudo GOPATH=/tmp/tmpGOPATH go get -u -v code.google.com/p/go.tools/cmd/vet
+	sudo GOPATH=/tmp/tmpGOPATH go get -u -v golang.org/x/tools/cmd/cover
+	sudo GOPATH=/tmp/tmpGOPATH go get -u -v golang.org/x/tools/cmd/vet
 	sudo rm -rf /tmp/tmpGOPATH
 
 setup-gotools:
-	go get -v github.com/golang/lint/golint
+	go get -u github.com/golang/lint/golint
+	go get -u golang.org/x/tools/cmd/cover
```

Comment:
It doesn't work for me to install `go cover` and `go vet` this way:
```console
$ go get -u golang.org/x/tools/cmd/cover
go install golang.org/x/tools/cmd/cover: open /usr/local/go/pkg/tool/darwin_amd64/cover: permission denied
```
That's why there is the temp GOPATH workaround in the setup-root-gotools target. Those tools are special and need to go in the root (default) GOPATH where go itself lives: https://groups.google.com/d/msg/golang-nuts/lz0nPiUwfUk/E92u9uZhMHYJ